accounts/migrations/0044_new-analytics-permissions.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-10-23 16:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("accounts", "0043_gestational-age-can-be-blank")]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="child",
+            options={
+                "ordering": ["-birthday"],
+                "permissions": [
+                    ("can_view_child_analytics_data", "Can View Child Analytics Data")
```

Comment:
Can the more human-readable name be something like Can View All Children from Analytics for clarity (since everyone can view child analytics data, the question is which children)

Reply:
That's a much better name - will do